src/setting_definitions/__init__.py

```diff
@@ -16,6 +16,7 @@ def vault_required(settings):
            or settings["certificate"] == "production" \
            or settings["certificate"] == "support" \
            or uses_vault_passwords \
+           or settings["notify_channel"] != "" \
```

Comment:
More pythonic to just write `or settings["notify_channel"]`?

Reply:
Pythons truthyness behaviour is so weird.  But you're right - I'll do that